torch/csrc/jit/passes/quantization.cpp

```diff
@@ -695,6 +695,32 @@ bool isWeightOfConvOrLinear(Value* v) {
   return result;
 }
 
+// Go through the CallMethod graph to check if the value is Weight.
+bool isWeight(Module& module, Value* v) {
+  if (isWeightOfConvOrLinear(v)) {
+    return true;
+  }
+  bool result = false;
+  auto* self = v->owningGraph()->inputs()[0];
+  for (const Use& u : v->uses()) {
+    Node* n = u.user;
+    if (n->kind() == prim::CallMethod) {
+      auto m = getInvokedModule(module, n, self);
+      std::shared_ptr<Graph> g = m.get_method(n->s(attr::name)).graph();
+      auto call_method_result = isWeight(m, g->inputs()[u.offset]);
+      if (!call_method_result) {
+        if (result) {
+          TORCH_CHECK(call_method_result == result,
+                      "Expected all CallMethods in the graph to produce same result.")
+        }
+      } else {
+        result = call_method_result;
+      }
```

Comment:
I don't think this works, what if the first CallMethod returns false, and second CallMethod returns true? you'll need to have an `optional<bool>` here I think